spec/chitter_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../lib/peep.rb'
+require 'database_helpers'
+
+describe Peep do
+  describe '#post' do
+    it 'saves and returns the posted peep' do
+      peep = Peep.post("Hello, Chitter!")
+      expect(peep.content).to eq ("Hello, Chitter!")
+      expect(peep.post_time).to eq Time.now.strftime("%H:%M:%S")
+    end
+  end
+
+  describe '#list' do
+    it 'lists all the posted peeps' do
+      
```

Comment:
It would be a more robust test to add a second peep to test, as the test wording includes 'lists all'.